### PR TITLE
AMP-28004 IATI support / troubleshooting for AMP Workshop

### PR DESCRIPTION
**Pull request by gmutuhu**
AMP-28004 IATI support / troubleshooting for AMP Workshop
The reported issue occurs when some field have no value mappings. I have modified the import to provide more information in the error message about the affected fields.

### CODE REVIEW

File: import-core/import-rest/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.devgateway.importtool</groupId>
 		<artifactId>importtool</artifactId>
-		<version>0.0.18-DEV-SNAPSHOT</version>
```
Comment (jdeanquin-dg) [inline import-core/import-rest/pom.xml]: DEV should remain in this PR. Actually it should be 19-snapshot. When we released 18 we forgot to upgrade 19

Comment (jdeanquin-dg) [reply import-core/import-rest/pom.xml]: same goes for all poms

Comment (gmutuhu) [reply import-core/import-rest/pom.xml]: Hey Julian, I noticed the version issue last week and created this pr: https://github.com/devgateway/iatiimport/pull/34.

**Comment by jdeanquin-dg** [comment]
please resolve the conflicts ( I merged the version increase)

**Pull request merged**